rfcs/react-components/non-core-components.md

```diff
@@ -0,0 +1,388 @@
+        - [Cons](#cons-4)
+    - [Package scope](#package-scope)
+      - [@fluentui scope](#fluentui-scope)
+        - [Pros](#pros-5)
+        - [Cons](#cons-5)
+      - [New package scope](#new-package-scope)
+        - [Pros](#pros-6)
+        - [Cons](#cons-6)
+    - [How users should consume components](#how-users-should-consume-components)
+      - [Suite packages](#suite-packages)
+        - [Pros](#pros-7)
+        - [Cons](#cons-7)
+      - [Individual packages](#individual-packages)
+        - [Pros](#pros-8)
+        - [Cons](#cons-8)
+    - [How to depend on core components](#how-to-depend-on-core-components)
+      - [Depend directly on components packages](#depend-directly-on-components-packages)
+        - [Pros](#pros-9)
+        - [Cons](#cons-9)
+      - [Depend on suite package is peer dependency](#depend-on-suite-package-is-peer-dependency)
+      - [Pros](#pros-10)
+      - [Cons](#cons-10)
+    - [Documentation](#documentation)
+      - [Current docsite](#current-docsite)
+        - [Pros](#pros-11)
+        - [Cons](#cons-11)
+      - [New docsite](#new-docsite)
+        - [Pros](#pros-12)
+        - [Cons](#cons-12)
+    - [Parnter ownership and collaboration](#parnter-ownership-and-collaboration)
+      - [GitHub](#github)
+        - [Pros](#pros-13)
+        - [Cons](#cons-13)
+      - [ADO mirroring](#ado-mirroring)
+        - [Pros](#pros-14)
+        - [Cons](#cons-14)
+      - [Both](#both)
+        - [Pros](#pros-15)
+        - [Cons](#cons-15)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# RFC: Support for components in the Fluent UI umbrella that are not core
+
+[@ling1726](https://github.com/ling1726) [@layershifter](https://github.com/layershifter)
+
+## Summary
+
+@fluentui/react-components has matured, with many flexible primitives that are also highly customizable. These
+primitive components generally don't reuse each other and are considered the 'building blocks' of a design system.
+Future design iterations might introduce components that are either:
+
+- Very situational - i.e. applied to specific scenarios only
+- Composed of multiple primitive components
+- Specializations of primitive components
+
+We need to formalise how we will deliver these components not only as first party packages but also second party
+packages with our partners who might need more situational components powered by V9. This RFC refers to these kinds
+of components as `non-core`.
+
+## Problem statement
+
+We create components that will not be shipped as a part of our 'core' components under the `@fluentui/react-components`
+library but other packages. We will have to finalize the following requirements:
+
+- Should non-core components be able to deviate from V9 core concepts?
+- Should non-core components have a faster breaking change cadence?
+- Should non-core components live in a separate repo?
+- Should non-core components still be under the `@fluentui` scope?
```

Comment:
this is a great question.....and if they are under this scope, how do we differentiate between them and core controls

other frameworks use 'contrib' to denote outside packages.

Reply:
contrib is a good example, I like that

Reply:
what other frameworks are you referring to ? I'm not aware about that pattern.

Reply:
yes they should, based on other popular OSS projects that's the common pattern. Only separation is some sort of pre-fix like:

- `@fluentui/lab-foo` ( experimental package )  
- `@fluentui/x-chat` ( extension package )